Feisty/Login/ViewController.swift

```diff
@@ -21,12 +21,22 @@ class ViewController: UIViewController {
 
   override func viewDidLoad() {
     super.viewDidLoad()
-    // Do any additional setup after loading the view.
+    
+    let colour = UIColor(red: 240, green: 240, blue: 240, alpha: 1)
```

Comment:
If you plan on using this colour in the future anywhere, rather define it somewhere

Reply:
Ah yes, it is only in the later stages that I though about some colours to the asset registry. This colour would the LightGrey from that same registry. I plan to make sure that all the code is prim and proper once I get images into the application.

Reply:
Thank you for the tip though ^^